[roslaunch][INFO] 2017-02-11 12:07:42,681: Checking log directory for disk usage. This may take awhile.
Press Ctrl-C to interrupt
[roslaunch][INFO] 2017-02-11 12:07:42,690: Done checking log file disk usage. Usage is <1GB.
[roslaunch][INFO] 2017-02-11 12:07:42,691: roslaunch starting with args ['/opt/ros/jade/bin/roslaunch', 'docker_ui', 'monitor.launch', 'video:=true', 'video_stream:=/bebop/image_raw']
[roslaunch][INFO] 2017-02-11 12:07:42,692: roslaunch env is {'LANG': 'en_US.UTF-8', 'ROS_LOG_FILENAME': '/root/.ros/log/b0d9e2bc-f052-11e6-a5ea-c0ffeec0ffee/roslaunch-moby-1.log', 'ROS_DISTRO': 'jade', 'ifaces': "['eno1']", 'SHLVL': '0', 'PYTHONPATH': '/home/root/catkin_ws/devel/lib/python2.7/dist-packages:/opt/ros/jade/lib/python2.7/dist-packages', 'ROS_IP': '192.168.201.26', 'HOSTNAME': 'moby', 'ROSLISP_PACKAGE_DIRECTORIES': '/home/root/catkin_ws/devel/share/common-lisp', 'ROS_ETC_DIR': '/opt/ros/jade/etc/ros', 'ROS_ROOT': '/opt/ros/jade/share/ros', 'no_proxy': '*.local, 169.254/16', 'PWD': '/', 'CPATH': '/home/root/catkin_ws/devel/include:/opt/ros/jade/include', 'ROS_MASTER_URI': 'http://192.168.201.4:11311', 'PATH': '/opt/ros/jade/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin', 'PKG_CONFIG_PATH': '/home/root/catkin_ws/devel/lib/pkgconfig:/opt/ros/jade/lib/pkgconfig', 'ROS_PACKAGE_PATH': '/home/root/catkin_ws/src:/opt/ros/jade/share:/opt/ros/jade/stacks', 'CMAKE_PREFIX_PATH': '/home/root/catkin_ws/devel:/opt/ros/jade', 'HOME': '/root', 'LD_LIBRARY_PATH': '/home/root/catkin_ws/devel/lib:/opt/ros/jade/lib'}
[roslaunch][INFO] 2017-02-11 12:07:42,692: starting in server mode
[roslaunch.parent][INFO] 2017-02-11 12:07:42,693: starting roslaunch parent run
[roslaunch][INFO] 2017-02-11 12:07:42,694: loading roscore config file /opt/ros/jade/etc/ros/roscore.xml
[roslaunch][INFO] 2017-02-11 12:07:42,755: Added core node of type [rosout/rosout] in namespace [/]
[roslaunch.config][INFO] 2017-02-11 12:07:42,757: loading config file /home/root/catkin_ws/src/ros_docker_ui/launch/monitor.launch
[roslaunch][INFO] 2017-02-11 12:07:42,758: Added node of type [diagnostic_common_diagnostics/tf_monitor.py] in namespace [/]
[roslaunch][INFO] 2017-02-11 12:07:42,772: Added node of type [rosbridge_server/rosbridge_websocket] in namespace [/]
[roslaunch][INFO] 2017-02-11 12:07:42,774: Added node of type [rosapi/rosapi_node] in namespace [/]
[roslaunch][INFO] 2017-02-11 12:07:42,777: Added node of type [roswww/webserver.py] in namespace [/]
[roslaunch][INFO] 2017-02-11 12:07:42,780: Added node of type [docker_ui/docker_ui_node.py] in namespace [/]
[roslaunch][INFO] 2017-02-11 12:07:42,781: Added node of type [web_video_server/web_video_server] in namespace [/]
[roslaunch][INFO] 2017-02-11 12:07:42,786: ... selected machine [] for node of type [diagnostic_common_diagnostics/tf_monitor.py]
[roslaunch][INFO] 2017-02-11 12:07:42,787: ... selected machine [] for node of type [rosbridge_server/rosbridge_websocket]
[roslaunch][INFO] 2017-02-11 12:07:42,788: ... selected machine [] for node of type [rosapi/rosapi_node]
[roslaunch][INFO] 2017-02-11 12:07:42,789: ... selected machine [] for node of type [roswww/webserver.py]
[roslaunch][INFO] 2017-02-11 12:07:42,789: ... selected machine [] for node of type [docker_ui/docker_ui_node.py]
[roslaunch][INFO] 2017-02-11 12:07:42,790: ... selected machine [] for node of type [web_video_server/web_video_server]
[roslaunch.pmon][INFO] 2017-02-11 12:07:42,794: start_process_monitor: creating ProcessMonitor
[roslaunch.pmon][INFO] 2017-02-11 12:07:42,795: created process monitor <ProcessMonitor(ProcessMonitor-1, initial daemon)>
[roslaunch.pmon][INFO] 2017-02-11 12:07:42,796: start_process_monitor: ProcessMonitor started
[roslaunch.parent][INFO] 2017-02-11 12:07:42,797: starting parent XML-RPC server
[roslaunch.server][INFO] 2017-02-11 12:07:42,797: starting roslaunch XML-RPC server
[roslaunch.server][INFO] 2017-02-11 12:07:42,798: waiting for roslaunch XML-RPC server to initialize
[xmlrpc][INFO] 2017-02-11 12:07:42,798: XML-RPC server binding to 0.0.0.0:0
[xmlrpc][INFO] 2017-02-11 12:07:42,799: Started XML-RPC server [http://192.168.201.26:35069/]
[xmlrpc][INFO] 2017-02-11 12:07:42,800: xml rpc node: starting XML-RPC server
[roslaunch.pmon][INFO] 2017-02-11 12:08:58,615: ProcessMonitor.shutdown <ProcessMonitor(ProcessMonitor-1, started daemon 139798309095168)>
[roslaunch.pmon][INFO] 2017-02-11 12:08:58,687: ProcessMonitor._post_run <ProcessMonitor(ProcessMonitor-1, started daemon 139798309095168)>
[roslaunch.pmon][INFO] 2017-02-11 12:08:58,688: ProcessMonitor._post_run <ProcessMonitor(ProcessMonitor-1, started daemon 139798309095168)>: remaining procs are []
[roslaunch.pmon][INFO] 2017-02-11 12:08:58,691: ProcessMonitor exit: cleaning up data structures and signals
[roslaunch.pmon][INFO] 2017-02-11 12:08:58,693: ProcessMonitor exit: pmon has shutdown
[roslaunch][ERROR] 2017-02-11 12:08:58,693: Unable to contact my own server at [http://192.168.201.26:35069/].
This usually means that the network is not configured properly.

A common cause is that the machine cannot ping itself.  Please check
for errors by running:

	ping 192.168.201.26

For more tips, please see

	http://www.ros.org/wiki/ROS/NetworkSetup

[roslaunch][ERROR] 2017-02-11 12:08:58,694: The traceback for the exception was written to the log file
[roslaunch][ERROR] 2017-02-11 12:08:58,698: Traceback (most recent call last):
  File "/opt/ros/jade/lib/python2.7/dist-packages/roslaunch/__init__.py", line 307, in main
    p.start()
  File "/opt/ros/jade/lib/python2.7/dist-packages/roslaunch/parent.py", line 268, in start
    self._start_infrastructure()
  File "/opt/ros/jade/lib/python2.7/dist-packages/roslaunch/parent.py", line 226, in _start_infrastructure
    self._start_server()
  File "/opt/ros/jade/lib/python2.7/dist-packages/roslaunch/parent.py", line 177, in _start_server
    self.server.start()
  File "/opt/ros/jade/lib/python2.7/dist-packages/roslaunch/server.py", line 405, in start
    """%(self.uri, p.hostname))
RLException: Unable to contact my own server at [http://192.168.201.26:35069/].
This usually means that the network is not configured properly.

A common cause is that the machine cannot ping itself.  Please check
for errors by running:

	ping 192.168.201.26

For more tips, please see

	http://www.ros.org/wiki/ROS/NetworkSetup


[rospy.core][INFO] 2017-02-11 12:08:58,699: signal_shutdown [atexit]
